documents/entityrelationship/repository.go

```diff
@@ -29,14 +31,56 @@ func newDBRepository(db storage.Repository, docRepo documents.Repository) reposi
 	return r
 }
 
-// Find returns a EntityRelationship based on a entity id and a targetDID
-func (r *repo) Find(entityIdentifier []byte, targetDID identity.DID) (*EntityRelationship, error) {
-	// todo not implemented
-	return nil, nil
+// Find returns the latest (second if revoked) version of an EntityRelationship based on a entity id and a targetDID
```

Comment:
The comment needs to be adjusted for the new purpose

Reply:
what new purpose?

Reply:
👍